setup.cfg

```diff
@@ -35,6 +35,8 @@ install_requires =
     astropy>=3.1
     requests
     PyYAML
+    photutils
```

Comment:
If you put these in `install_requires`, they are not needed in `extras_require`.

You should think about whether these are really required at install time (`install_requires`) or they are merely optional dependencies that are only needed if one needs to use one small part of the package (`extras_require`).

Reply:
Optional dependencies should only be imported locally when needed and nothing should crash horribly outside of the things that actually use them when they are not present.

Reply:
Given that these are used at only specific times in the pipeline, I moved them to extras_require.

Reply:
I still see them listed here. Did you forget to remove?

Reply:
Yes, I just forgot to commit. Fixed